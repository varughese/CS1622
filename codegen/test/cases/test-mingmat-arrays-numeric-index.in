int garr[15];

void main(void) {
    int x[30];
    x[6] = 6000;
    garr[1] = 100;
    garr[3] = 300;
    output(garr[1]);
    output(garr[3]);
    output(x[6]);
    {
        int x[10];
        int y[200];
        x[6] = 60000;
        y[4] = 40000;
        output(x[6]);
        output(y[4]);
    }
    /* garr[3+2] = 696969;*/
}