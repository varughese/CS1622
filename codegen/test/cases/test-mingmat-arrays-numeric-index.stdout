100
300
6000
60000
40000